src/pages/explorer/index.tsx

```diff
@@ -26,3 +29,31 @@ export const ExplorerPage = () => {
     </Page>
   );
 };
+
+const ExplorerEvents = () => {
+  const [mounted, setMounted] = useState(false);
```

Comment:
probably no real reason to check if mounted

Reply:
You need it if for type change and filter change. When you arrive in the page you should only trigger search event. Type and filter should be triggered only when they change from one state to the next, the initial state doesn't count as the search already send it to analysis